Homepage.

YuriGizra opened the pull request:
#195
Output:
![selection_336](https://cloud.githubusercontent.com/assets/7760669/11065209/b218eb6a-87ca-11e5-9bcc-92b1100355b8.jpg)

YuriGizra commented:
Added our works :
![selection_340](https://cloud.githubusercontent.com/assets/7760669/11093242/ccc3c616-8891-11e5-866f-8af558591587.jpg)

YuriGizra commented:
Imtarmediate state: 
![selection_342](https://cloud.githubusercontent.com/assets/7760669/11098107/741ebb00-88ab-11e5-957c-ae84066aaded.jpg)

YuriGizra commented:
Carousel markup:
![selection_344](https://cloud.githubusercontent.com/assets/7760669/11115978/c5d2dbde-8937-11e5-8a4f-0fbd55bb5ace.jpg)

YuriGizra commented:
add quote marks:
![selection_346](https://cloud.githubusercontent.com/assets/7760669/11120492/be80d90a-8958-11e5-83f6-080bea5ff512.jpg)

About us section pixel perfect:
![selection_345](https://cloud.githubusercontent.com/assets/7760669/11120481/a3e60c46-8958-11e5-8cc6-460530f7b2ef.jpg)

amitaibu commented:
:+1:

YuriGizra commented:
Add blog section:
![Uploading Selection_347.jpg…]()

amitaibu commented:
Image is missing

YuriGizra commented:
Footer:
![selection_348](https://cloud.githubusercontent.com/assets/7760669/11168270/3d4e7c54-8b90-11e5-893d-ccc30c4af28e.jpg)

YuriGizra commented:
@amitaibu, ready for review.

amitaibu commented:
Please update visual regression test, so it's green

YuriGizra commented:
@amitaibu  Removed the Visual regression.

amitaibu commented:
Can you add layered output images please

YuriGizra commented:
Header layered:
![selection_349](https://cloud.githubusercontent.com/assets/7760669/11169259/3e1fd60c-8bbb-11e5-82a6-5524ede18dfd.jpg)
output:
![selection_350](https://cloud.githubusercontent.com/assets/7760669/11169263/494c65f4-8bbb-11e5-99fa-d00300d52a13.jpg)

responces:
![selection_351](https://cloud.githubusercontent.com/assets/7760669/11169289/f9ba7a2a-8bbb-11e5-9425-d19f8195e5a4.jpg)

![selection_353](https://cloud.githubusercontent.com/assets/7760669/11169290/ff044498-8bbb-11e5-81b5-bca411545d7e.jpg)

About us:
![selection_354](https://cloud.githubusercontent.com/assets/7760669/11169292/0759cd7a-8bbc-11e5-8f23-57e6f25333bd.jpg)

![selection_355](https://cloud.githubusercontent.com/assets/7760669/11169293/0be603a4-8bbc-11e5-896f-6ad4de41c63f.jpg)

from blog:
![selection_356](https://cloud.githubusercontent.com/assets/7760669/11169294/1e1357c0-8bbc-11e5-8fed-bceedead8910.jpg)

![selection_357](https://cloud.githubusercontent.com/assets/7760669/11169295/21c4a8c4-8bbc-11e5-8be9-496f0b820951.jpg)

Footer:
![selection_358](https://cloud.githubusercontent.com/assets/7760669/11169296/271e9af0-8bbc-11e5-8e24-15712633c3e1.jpg)

![selection_359](https://cloud.githubusercontent.com/assets/7760669/11169298/2c25c136-8bbc-11e5-81af-8714313c9012.jpg)

bricel commented:
Please add fax number to footer as well.

amitaibu commented:
@bricel we decided not to add the fax on account that it looks "old"

bricel commented:
What the fax ! :-) It's actually still useful, we could maybe hide it somehow ?

YuriGizra commented:
@amitaibu, @bricel,
So, should I add fax number  or no?

amitaibu commented:
Not needed

YuriGizra commented:
ok. It's ready for review.

amitaibu commented:
Needs more work :)

![your_project__the_gizra_way](https://cloud.githubusercontent.com/assets/125707/11177032/27b74c52-8c4b-11e5-978d-2a3d54fcad06.jpg)

amitaibu commented:
![your_project__the_gizra_way-3](https://cloud.githubusercontent.com/assets/125707/11177054/557371fc-8c4b-11e5-913a-6316e8fc87c9.jpg)

YuriGizra commented:
* [ ] 1200 px container width for all screen  except mobile (<767px).
* [ ] Fonts.
* [ ] Add Logo Acquila.
* [ ] header / footer  backgroun min-width 1200px.
* [ ] Add links in blog sections.

YuriGizra commented:
Added font Lato. It is closest to GillSans
Fixed fonts:
added links to blog section.

![selection_362](https://cloud.githubusercontent.com/assets/7760669/11179608/4ee3fc64-8c5d-11e5-9f1f-302fa3423612.jpg)

![selection_363](https://cloud.githubusercontent.com/assets/7760669/11179611/563fa404-8c5d-11e5-8095-da1d2e71cd2a.jpg)

![selection_360](https://cloud.githubusercontent.com/assets/7760669/11179618/5c4fea84-8c5d-11e5-924e-5a7e2f38b024.jpg)

amitaibu commented:
:+1:

YuriGizra commented:
Fixed container size for all screen except mobile:
Output:
![selection_364](https://cloud.githubusercontent.com/assets/7760669/11182639/dbb00d6a-8c74-11e5-88aa-b74b32b50576.jpg)

Very high resolution screen:
![selection_365](https://cloud.githubusercontent.com/assets/7760669/11182611/b2d2f6d2-8c74-11e5-972f-b417e484af21.jpg)

![selection_366](https://cloud.githubusercontent.com/assets/7760669/11182644/e666fb4c-8c74-11e5-858e-2ee30fb7d119.jpg)

YuriGizra commented:
Added logo. Fixed according design:
![selection_379](https://cloud.githubusercontent.com/assets/7760669/11323708/cbba0782-9122-11e5-896c-58d621960287.jpg)
![selection_380](https://cloud.githubusercontent.com/assets/7760669/11323722/1f7df3d8-9123-11e5-9af8-d020d6adb914.jpg)

![selection_381](https://cloud.githubusercontent.com/assets/7760669/11323724/2480842c-9123-11e5-83f0-27d16f8a4ec6.jpg)

![selection_382](https://cloud.githubusercontent.com/assets/7760669/11323725/28666d9a-9123-11e5-8f10-575f30bc7ecb.jpg)

YuriGizra commented:
Added popup block.
![acquia](https://cloud.githubusercontent.com/assets/7760669/11324850/a053afac-9146-11e5-895c-d9529b892ef8.gif)

amitaibu commented:
Can you update the READMR with the installation needed (`gem update --system && gem install scss-lint`)

Also, I'm getting this error on `grunt serve`

```
    loading tasks  1s  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 99%
    Total 1s
        Warning: Running "bootlint:files" (bootlint) task
    Warning: Object #<Object> has no method 'forEach' Use --force to continue.
    
    Aborted due to warnings.
```

YuriGizra commented:
@amitaibu, Ready for review.

amitaibu commented on package.json:
```diff
@@ -27,7 +27,8 @@
     "load-grunt-tasks": "~0.2.0",
-    "time-grunt": "~0.2.8"
+    "time-grunt": "~0.2.8",
+    "grunt-scss-lint": "~0.3.8"
```
is this still used?

YuriGizra replied:
Yes, scss-lint is working.

amitaibu commented on .scss-lint.yml:
```diff
@@ -0,0 +1,31 @@
+# Configuration scss-lint for Gizra.com.
+
+scss_files: "app/_scss/*.scss"
```
is this still used? Maybe worth to move to own PR.

YuriGizra commented:
Added hover intent:
[link to video](http://gfycat.com/FrequentQuarrelsomeAmericanbittern)

Added links to projects:
* [harvard](http://openscholar.harvard.edu)
* [acquia](http://www.acquia.com)
* [commerce-guys](http://commerceguys.com)
* [european commision: c4d ](http://capacity4dev.ec.europa.eu)

@amitaibu, What link will be right for EC?
* [grammy](http://www.grammy.com)
* [wix](http://www.wix.com)
* [whitehouse](http://www.whitehouse.gov)

amitaibu commented:
We will add the links later - for now it can be a placeholder

amitaibu commented:
@YuriGizra please assign to me when ready for review. Also, please update title, and add a screenshot of the banner, to see the background is in place.

YuriGizra commented:
Fixed background header:
![selection_387](https://cloud.githubusercontent.com/assets/7760669/11337459/e0358c6c-91f5-11e5-903d-c66a05eec5b6.jpg)

YuriGizra commented:
Accidentally closed.

YuriGizra commented:
jekyll recent post:
![selection_389](https://cloud.githubusercontent.com/assets/7760669/11339002/ecc82296-91ff-11e5-92c4-5268d9faa210.jpg)

@amitaibu, 
Should inner links  be the same color? 
![selection_388](https://cloud.githubusercontent.com/assets/7760669/11339076/56243248-9200-11e5-8cec-057bc3a24be7.jpg)

amitaibu commented:
Lets take the first 300 words, and strip tags (i.e. no images, and no links)

YuriGizra commented:
Stripped and  truncated:
![selection_390](https://cloud.githubusercontent.com/assets/7760669/11340473/b068c8d0-9206-11e5-9165-9a14d09d7929.jpg)

Centered hover [block](http://www.gfycat.com/FirstNeighboringAntlion)

YuriGizra commented:
Fixed title font:
![selection_404](https://cloud.githubusercontent.com/assets/7760669/11393315/b231fe7c-9367-11e5-9101-1c13c94ae08c.jpg)

YuriGizra commented:
@amitaibu,
Ready for review.

amitaibu commented:
@YaronMiro please review

YaronMiro commented on app/index.html:
```diff
@@ -5,84 +5,156 @@
+    <div class="row">
+      <div class="col-sm-12">
+        <nav id="nav" role="navigation" class="navbar navbar-default">
+          <div class="container-fluid">
```
@YuriGizra 
Is the `container-fluid` class really needed ?

YaronMiro commented on app/index.html:
```diff
@@ -5,84 +5,156 @@
+          </div>
+        </nav>
+        <div id="slogan">
+          <div class="gizra-logo"><a href="/">gizra</a></div>
```
Please replace (Better for seo)
`<div class="gizra-logo">` => `<h1 class="gizra-logo">`

YaronMiro replied:
I will fix this ^^ :)

YaronMiro commented on app/index.html:
```diff
@@ -90,34 +162,63 @@ <h2 class="title abril-font">You might have seen our work already</h2>
+        <h2 class="">From our blog:</h2>
+        {% for post in site.posts limit:1 %}
+          {% if post.author == null %}
+          {% assign author = "Amitaibu" %}
```
@YuriGizra 
Please indent so it is easy to read.
```
{% if post.author == null %}
     {% assign author = "Amitaibu" %}
{% else %}
     {% assign author = post.author %}
 {% endif %}
```
to the same on the rest of the jekyll code

YaronMiro replied:
I will fix this ^^ :)

YaronMiro commented:
Thanks Merged manually.
